homework/07LessonSeven/script.js

```diff
@@ -13,12 +13,12 @@ $(document).on('ready', function() {
   // Problem 1:
   // select the h1 element
   // assign the selected jQuery object to the property window.h1
-  window.h1;
+  window.h1 = $('h1');
```

Comment:
Hi Carla! Looks like the homework isn't passing the tests. Is it passing on your end? We may need to take a look on Wednesday.